wled00/wled.h

```diff
@@ -504,6 +504,8 @@ WLED_GLOBAL byte statusESPNow        _INIT(ESP_NOW_STATE_UNINIT); // state of ES
 WLED_GLOBAL bool useESPNowSync       _INIT(false);  // use ESP-NOW wireless technology for sync
 WLED_GLOBAL char linked_remote[13]   _INIT("");     // MAC of ESP-NOW remote (Wiz Mote)
 WLED_GLOBAL char last_signal_src[13] _INIT("");     // last seen ESP-NOW sender
+WLED_GLOBAL byte channelESPNow       _INIT(1);      // last channel used when searching for master
```

Comment:
per my comment I would suggest having a `last_signal_channel` and when it's non-zero, change the scanning algorithm a bit.